packages/terra-form-select/src/Combobox.jsx

```diff
@@ -8,7 +8,7 @@ import SelectUtil from './shared/_SelectUtil';
 
 const propTypes = {
   /**
-   * Whether a clear option is available to clear the selection, will use placeholder text if provided.
+   * Whether a clear option is available to clear the selection, will use placeholder text.
```

Comment:
Could you add what the provided placeholder text is here?

Reply:
Updated `allowClear` description to be -
`Whether a clear option is available to clear the selection, will use  - Select - as the clear option.`